tests/unit/analysis/test_sweep.py

```
@@ -0,0 +1,1570 @@
+        start = np.array([0.0, np.nan])
+        gen_start = np.array([10.0, np.nan])
+        end = np.array([110.0, 120.0])
+        input_tok = np.array([100.0, np.nan])
+        output_tok = np.array([20.0, 40.0])
+
+        # req0: 2 chunks of 50ns; req1 has ICL entries but NaN gen_start
+        icl_vals = np.array([50.0, 50.0, 30.0, 30.0], dtype=np.float64)
+        icl_rec = np.array([0, 0, 1, 1], dtype=np.int32)
+        icl_off = np.array([0, 2], dtype=np.int64)
+
+        ts, tif = tokens_in_flight_sweep_line_icl(
```

Ruff (0.15.20)

[warning] Unpacked variable `ts` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)